index.js

```diff
@@ -2,7 +2,6 @@
 
 const anymatch = require("anymatch")
 const ngAnnotate = require("ng-annotate")
-const assignToCopy = (...sources) => Object.assign(...sources)
```

Comment:
Actually this was meant to say `(...sources) => Object.assign({}, ...sources)`. Could you update your calls to add an empty object as the first argument?